tests/suite/custom_resources_utils.py

```diff
@@ -423,6 +564,32 @@ def delete_virtual_server(custom_objects: CustomObjectsApi, name, namespace) ->
     print(f"VirtualServer was removed with name '{name}'")
 
 
+def delete_transport_server(custom_objects: CustomObjectsApi, name, namespace) -> None:
```

Comment:
do we need both this one and `delete_ts`?

Reply:
removed `delete_transport_server`